core/src/test/java/kafka/test/ClusterConfigTest.java

```diff
@@ -96,4 +99,16 @@ public void testDisksPerBrokerIsZero() {
                 .setDisksPerBroker(0)
                 .build());
     }
+
+    @Test
+    public void testDisplayTags() {
+        List<String> tags = Collections.singletonList("example tag");
+        ClusterConfig clusterConfig = ClusterConfig.defaultBuilder().setTags(tags).build();
+
+        Set<String> expectedDisplayTags = new LinkedHashSet<>(tags);
```

Comment:
This testing code is almost same with production code. Maybe we don't need to verify the "format" of display tags. Instead, we check the output which should "contain" expected strings. WDYT?